docs/how-tos/howto-add-new-validators.md

```diff
@@ -0,0 +1,56 @@
+---
+id: howto-add-new-validators
+title: How to add new validators in a network
+---
+## Requirements
+
+### Setup IBFT
+
+To setup the network with 4 active validator nodes localy is covered in the [setup IBFT locally](/docs//how-tos/howto-setup-ibft/howto-set-ibft-locally) section.
+
+## Overview
+This guide goes into detail on how to add a new validator node to an active IBFT network with 4 validator nodes.
+
+## Step 1: Initialize data folders for IBFT and generate validator keys​ for the new node
+
+In order to get up and running with IBFT, you need to initialize the data folders, for the new node:
+
+````bash
+go run main.go secrets init --data-dir test-chain-5
+````
+
+This command will print the validator key (address) and the node ID. You will need the validator key (address) for the next step.
+
+## Step 2: From other validator nodes proposes a new candidate to be added
+
+Other candidates need to propose a new node to become a validator. 
+
+Example of how to validator node on grpc address: 127.0.0.1:10000 to propose node with validator key (address): 0x8B15464F8233F718c8605B16eBADA6fc09181fC2:
```

Comment:
```suggestion
Example of how to propose a new validator (`0x8B15464F8233F718c8605B16eBADA6fc09181fC2`) from the existing validator node on grpc address: 127.0.0.1:10000:
```

Reply:
Also add information on how this needs to be done on at least 51% of all validators.